widgetssdk/src/main/java/com/glia/widgets/chat/domain/gva/MapGvaResponseTextUseCase.kt

```diff
@@ -1,23 +1,21 @@
 package com.glia.widgets.chat.domain.gva
 
-import com.glia.widgets.chat.model.ChatState
 import com.glia.widgets.chat.model.Gva
 import com.glia.widgets.chat.model.GvaResponseText
 import com.glia.widgets.core.engagement.domain.model.ChatMessageInternal
-import java.util.UUID
 
 internal class MapGvaResponseTextUseCase {
-    operator fun invoke(chatMessage: ChatMessageInternal, chatState: ChatState): GvaResponseText {
+    operator fun invoke(chatMessage: ChatMessageInternal, showChatHead: Boolean): GvaResponseText {
         val message = chatMessage.chatMessage
 
         return GvaResponseText(
             id = message.id,
             content = message.metadata?.optString(Gva.Keys.CONTENT).orEmpty(),
-            showChatHead = false,
-            operatorId = chatMessage.operatorId ?: UUID.randomUUID().toString(),
+            showChatHead = showChatHead,
```

Comment:
The GVA usecases are really similar. I understand the slightly differing context, but it bothers me a little